package.json

```diff
@@ -35,21 +35,25 @@
     "/dist",
     "/lib"
   ],
+  "resolutions": {
+    "bn.js": "4.12.0",
```

Comment:
Are these resolutions still needed?

Reply:
I ask because these resolutions will only get applied here, not by users of this package. As such I usually only see them being used in projects or libraries that produce a bundle, not in regular npm libraries like this.

Reply:
I tend to think it's better not to include them in cases like this because it can be misleading. But I don't feel strongly enough for it to be a blocker